alcs-frontend/src/app/features/board/dialogs/planning-review/planning-review-dialog.component.ts

```diff
@@ -40,11 +44,38 @@ export class PlanningReviewDialogComponent extends CardDialogComponent implement
     super.ngOnInit();
 
     this.planningReview = this.data;
-    this.populateCardDate(this.data.card);
+    this.populateCardData(this.data.card);
 
     this.selectedRegion = this.data.region.code;
     this.cardTitle = `${this.data.fileNumber} (${this.data.type})`;
 
     this.title = this.planningReview.fileNumber;
   }
+
+  private async reload() {
+    const noticeOfIntent = await this.planningReviewService.fetchByCardUuid(this.planningReview.card.uuid);
+    if (noticeOfIntent) {
+      this.populateCardData(noticeOfIntent.card);
+    }
+  }
+
+  async onBoardSelected(board: BoardWithFavourite) {
+    this.selectedBoard = board.code;
+    try {
+      await this.boardService.changeBoard(this.planningReview.card.uuid, board.code);
+      const loadedBoard = this.boards.find((board) => board.code === this.selectedBoard);
+      if (loadedBoard) {
+        this.boardStatuses = loadedBoard.statuses;
+      }
+
+      this.isDirty = true;
+      const toast = this.toastService.showSuccessToast(`Planning Review moved to ${board.title}`, 'Go to Board');
+      toast.onAction().subscribe(() => {
+        this.router.navigate(['/board', board.code]);
+      });
+      await this.reload();
```

Comment:
Is this only applicable to NOI and not any other card type?

Reply:
This change is in the Planning Review Dialog, nothing to do with NOI's.

Reply:
I got confused by reload() itself: 
"if (noticeOfIntent) {
      this.populateCardData(noticeOfIntent.card);
    }"